### ISSUE TITLE
Is there an example about regression task for texts with keras?

### ISSUE THREAD

**Issue by Imorton-zd** [fchollet/keras#2207]

**Comment by jrieke** [comment]
https://github.com/fchollet/keras/blob/master/examples/imdb_lstm.py
This one is for binary classification (sentiment analysis), but it should be quite similar for multiple categories if you use a softmax instead of the sigmoid layer and the `categorical_crossentropy` loss function.

**Comment by Imorton-zd** [comment]
@jrieke Thanks for your reply! I am afraid that my description is ambiguous. I want to know if there is an example to do regression task not classification.

**Comment by rpinsler** [comment]
Use a linear activation function at the last layer and MSE as your loss function. Most people also seem to use rmsprop as optimizer for regression tasks.

**Comment by Imorton-zd** [comment]
@rpinsler Thanks for your reply!

**Comment by cj-suda** [comment]
@rpinsler  I try what you said. But the result of model.predict() is same to result of model.predict_proba() like [ 0.08436067  0.06973539 -0.03013037 ...,  0.06828094  0.06003802
   0.29175624] which is a output of a test sample. The range of regression is [0,9], apparently the result is not we want. So what's wrong with that we have done?and how to get correct output.
